pyproject.toml

```diff
@@ -56,12 +56,18 @@ dev = [
   "arize[AutoEmbeddings, LLM_Evaluation]",
 ]
 experimental = [
-  "langchain>=0.0.257",
-  "llama-index>=0.8.25",
   "openai",
   "tenacity",
 ]
 
+llama_index = [
+  "llama-index>=0.8.25",
```

Comment:
```suggestion
  "openai",
  "llama-index>=0.8.25",
```

Reply:
@RogerHYang why is OpenAI needed?

Reply:
we have a line that says `if isinstance(raw, OpenAIObject):` in there

Reply:
maybe we can try to remove that line later

Reply:
https://github.com/Arize-ai/phoenix/issues/1388